public/src/components/channelManagement/testEditorArticleCountEditor.tsx

```diff
@@ -24,13 +24,15 @@ const styles = ({ spacing }: Theme) =>
       },
     },
     formContainer: {
+      maxWidth: '250px',
+
       '& > * + *': {
         marginTop: spacing(1),
       },
     },
   });
 
-const DEFAULT_ARTICLES_VIEWED_SETTINGS: ArticlesViewedSettings = {
+export const DEFAULT_ARTICLES_VIEWED_SETTINGS: ArticlesViewedSettings = {
   minViews: 3,
   maxViews: 999,
```

Comment:
can this be undefined by default? This field should be optional

Reply:
Both `minViews` and `maxViews` are optional - should we set both to undefined? or just `maxViews`?

Reply:
Let's give `minViews` a default as it's the one that's always used

Reply:
sure thing!